Bottom: 7c5c2840404d80327e2ec19554e8e75022aadbce
Top:    d053814133a0ce3f1ac016d88daab84bd32d3a8e
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-26 15:51:46 +0100

Cure crosshair lagging on screen

FIXME: SHOULD NOT RE-WRITE THE GTK HID


---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index d3d5c16..37dd2af 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -674,9 +674,7 @@ ghid_port_window_motion_cb (GtkWidget * widget,
   queue_tooltip_update (out);
 #endif
 
-  ghid_show_crosshair (FALSE);
-  ghid_show_crosshair (TRUE);
-  if (moved && have_crosshair_attachments ())
+  if (moved)
     ghid_draw_area_update (gport, NULL);
   return FALSE;
 }
